CMakeLists.txt

```diff
@@ -7,7 +7,7 @@ find_package(ament_cmake REQUIRED)
 include_directories(include)
 
 if(NOT WIN32)
-  set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -Wall -Wextra")
+  set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -Wall -Wextra -Wpedantic")
   # The warning flags are added per target, because otherwise the local
   # gmock library that is built will have compiler warnings.
   set(rcutils_extra_warning_flags "-Wall -Wextra -Wpedantic")
```

Comment:
I know you didn't add this line of code, but I'm not sure how to square the comment about gmock with the addition of -Wpedantic to the overall CMAKE_C_FLAGS.  Do you understand what that comment is talking about, and why we need `rcutils_extra_warning_flags` ?